spec/factories/event_schedule.rb

```diff
@@ -0,0 +1,22 @@
+# Read about factories at https://github.com/thoughtbot/factory_girl
+
+FactoryGirl.define do
+  factory :event_schedule do
+    event
+    after(:build) do |event_schedule|
```

Comment:
`room` and `start_time` are compulsory, and I have to create a `venue` before creating rooms. I also need to check if a room or date has been created already before creating it.
And always I create an `event_schedule` is because I want to schedule the event so I need that it belongs to the selected schedule, which must be created if it doesn't exit.

I could eliminate the part of the selected_schedule, but then I will have to do it every time I create an `event_schedule`, as in everywhere I use this factory this has to be done. I think that would be worst.

The rest was already on event and I just moved it (I kept the creation of the venue in both places but maybe it is not needed in event):

https://github.com/openSUSE/osem/pull/1094/files#diff-9c3fa59ac2be2f7c567e2bef4ec1367dL29

https://github.com/openSUSE/osem/pull/1094/files#diff-9c3fa59ac2be2f7c567e2bef4ec1367dL36